lib/bitbucket_rest_api/repos/webhooks.rb

```diff
@@ -51,7 +51,7 @@ def list(user_name_or_project_key, repo_name)
       url = if BitBucket.options[:bitbucket_server]
               "/1.0/projects/#{user_name_or_project_key}/repos/#{repo_name}/webhooks"
             else
-              "/2.0/repositories/#{user_name_or_project_key}/#{repo_name}/hooks"
+              "/2.0/repositories/#{user_name_or_project_key}/#{repo_name}/hooks?pagelen=100"
```

Comment:
This gem does not support passing `pagelen` into this call. Rather than spending time re-engineer the library, I opted to just increase the length of the page.